do-app-platform-op-cli/README.md

```diff
@@ -1,314 +1,285 @@
+#### 5.1: Get your Google service account key
+
+Follow the steps to [create a Google service account, key, and API client](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client), then [save the file](https://support.1password.com/files/) to the 1Password vault you created in step 2.1. Name the item `workspace-credentials`.
+
+#### 5.2: Download and edit the Google Workspace settings template
+
+Copy and paste the following command for your shell to [read the file using its secret reference](https://developer.1password.com/docs/cli/reference/commands/read), encode the credentials, and store them as a new field in the `workspace-credentials` item you just saved in your 1Password account. If you gave the vault a custom name, replace `op-scim` with the name of your vault, then run the command.
+
+**Bash**:
 
-  Using Bash:
   ```sh
-  op document create "pathToFile/workspace-credentials.json" --title "workspace-credentials" --vault "op-scim"
   op item edit "workspace-credentials" --vault "op-scim" base64_encoded_credentials=$(op read "op://op-scim/workspace-credentials/workspace-credentials.json" | base64 | tr -d "\n")
   ```
 
-  Using Powershell:
+**Powershell**:
+
   ```pwsh
-  op document create "pathToFile/workspace-credentials.json" --title "workspace-credentials" --vault "op-scim"
   op item edit "workspace-credentials" --vault "op-scim" base64_encoded_credentials=$([Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($(op read "op://op-scim/workspace-credentials/workspace-credentials.json"))))
   ```
 
-  3. Download and open the  [`workspace-settings.json`](/do-app-platform-op-cli/google-workspace/workspace-settings.json)) file in a text editor and replace the values for each key:
+#### 5.3: Download and edit the Google Workspace settings template
 
-  - `actor`: the email address for the administrator that the service account is acting on behalf of
-  - `bridgeAddress`: the URL you will use for your SCIM bridge (not your 1Password account sign-in address). This is the Application URL for your Container App found on the overview page. For example: https://op-scim-bridge-example.ondigitalocean.app
+1. Download the [`workspace-settings.json`](./google-workspace/workspace-settings.json) file from this repo.
+2. Edit the following in this file:
+	- **Actor**: Enter the email address of the Google Workspace administrator for the service account.
+	- **Bridge Address**: Enter your SCIM bridge domain. Used in [Step 4](#step-4-test-your-scim-bridge) to test your SCIM bridge. This is not your 1Password account sign-in address. For example:  `https://op-scim-bridge-example.ondigitalocean.app`.
```

Comment:
To match the capitalization in the other place where we referred to a step. And a contraction.

```suggestion
	- **Bridge Address**: Enter your SCIM bridge domain. Used in [step 4](#step-4-test-your-scim-bridge) to test your SCIM bridge. This isn't your 1Password account sign-in address. For example:  `https://op-scim-bridge-example.ondigitalocean.app`.
```